ogn/parser/parse.py

```diff
@@ -6,16 +6,18 @@
 from ogn.parser.exceptions import AprsParseError, OgnParseError
 
 
-def parse_aprs(message, reference_date=None):
+def parse_aprs(message, reference_date=None, reference_time=None):
     if reference_date is None:
```

Comment:
If we call createTimestamp(message, date, None) then message gets a timestamp with the date.
If we call createTimestamp(message, date, time) then message gets a timestamp with the corrected date.
So if we set a reference_time from None to utcnow().time the function createTimestamp will make the corrected date. A logfile (25.09.) with a entry at 00:01 imported at 23:59 will create a Timestamp 26.09. 00:01... Not good. Or did I miss the point?